frontend/src/hooks/api/secretSharing/types.ts

```diff
@@ -26,6 +33,21 @@ export type TCreateSharedSecretRequest = {
   accessType?: SecretSharingAccessType;
 };
 
+export type TCreateSecretRequestRequest = {
```

Comment:
I think the current convention for request types is for it to be prefixed with DTO?